docs/quickstart/example.py

```diff
@@ -31,8 +31,16 @@ class MySG(StatesGroup):
 
 @dp.message(Command("start"))
 async def start(message: Message, dialog_manager: DialogManager):
+    # Important: always set `mode=StartMode.RESET_STACK` you don't want to stack dialogs
     await dialog_manager.start(MySG.main, mode=StartMode.RESET_STACK)
 
 
 if __name__ == '__main__':
     dp.run_polling(bot, skip_updates=True)
+
+from aiogram_dialog import Window
```

Comment:
why is it here?

Reply:
because the imports in individual examples and in the finished one are different, should I move this to a another file?